### ISSUE TITLE
The Surprising Effectiveness of Linear Unsupervised Image-to-Image Translation 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#8]
# TL;DR
Domain transformation by GAN relies on localized bias, and even simple entire image change tasks (e.g., flipping up and down) fail. They found that even linear transformations alone can work well for those tasks if the transformation is done in a way that does not depend on localization (e.g., PCA). 	
![image](https://user-images.githubusercontent.com/24524018/89274575-15e77300-d67c-11ea-98ad-1399d8dbc2f1.png)

![image](https://user-images.githubusercontent.com/24524018/89274584-1a139080-d67c-11ea-80b2-a4a9aca99e7a.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.12568 

# Submission Dates(yyyy/mm/dd)
2020/07/24 	

# Authors and institutions
Eitan Richardson, Yair Weiss 
- The Hebrew University of Jerusalem

# Methods
Use PCA for unsupervised image translation. It takes only few seconds while GAN takes few hours or days.
![image](https://user-images.githubusercontent.com/24524018/89274667-331c4180-d67c-11ea-9c5b-dbd11bb5081f.png)

# Results

# Comments
